### PR TITLE
feat(profile): send push notifications after a profile update

### PR DESCRIPTION
**Pull request by eoger**
Connects to https://github.com/mozilla/fxa-features/issues/118

Disregard the structural changes in the JSON schema.

### CODE REVIEW

**Comment by eoger** [comment]
This can now be end-to-end tested using https://github.com/mozilla/fxa-profile-server/pull/241 and [Bugzilla #1345955](https://bugzilla.mozilla.org/show_bug.cgi?id=1345955)

File: config/index.js [hunk 1]
```diff
@@ -378,6 +378,20 @@ var conf = convict({
...
   },
+  profileServerMessaging: {
+    region: {
+      doc: 'The region where the queues live',
+      format: String,
+      env: 'PROFILE_MESSAGING_REGION',
+      default: ''
+    },
+    profileUpdatesQueueUrl: {
+      doc: 'The bounce queue URL to use (should include https://sqs.<region>.amazonaws.com/<account-id>/<queue-name>)',
```
Comment (rfk) [inline config/index.js]: nit: "bounce queue" is copypasta from some other config item?

**Comment by rfk** [comment]
> Disregard the structural changes in the JSON schema.

Is this waiting for some stuff to merge back from private repo?

File: config/index.js (hunk 1 above)
Comment (eoger) [inline config/index.js]: Yes, my bad.

**Comment by eoger** [comment]
Yes

**Comment by rfk** [comment]
IIUC the private branch has been deployed, I'll see about merging it to public

**Comment by eoger** [comment]
Comments addressed.
ajv's removeAdditional is borderline broken (which is why the tests previously failed), let's get it over with and reject misbehaving clients instead of trying to do clever tricks.

**Pull request merged**